aws/components/sqs/setup.ftl

```diff
@@ -71,6 +71,10 @@
                 name=dlqName
                 retention=1209600
                 receiveWait=20
+                tags=
+                    getOccurrenceCoreTags(
+                        occurrence,
+                        formatComponentFullName(core.Tier, core.Component, dlqName))
```

Comment:
Might as well keep the name the same as the one used above to name the resource

```suggestion
                        dlqName
```

Reply:
Cool, done